openwisp_firmware_upgrader/api/views.py

```diff
@@ -67,6 +68,20 @@ class BuildDetailView(OrgAPIMixin, generics.RetrieveUpdateDestroyAPIView):
     organization_field = 'category__organization'
 
 
+class BuildMassUpgradeView(OrgAPIMixin, generics.GenericAPIView):
+    model = Build
+    queryset = Build.objects.all().select_related('category')
+    serializer_class = BuildSerializer
+    lookup_fields = ['pk']
+    organization_field = 'category__organization'
+
+    def post(self, request, pk):
+        upgrade_all = request.POST.get('upgrade_all')
```

Comment:
I suggest to make this line more explicit:
```python
upgrade_all = request.POST.get('upgrade_all') is not None
```
So we get either `True` or `False`